pkg/controller/api_test.go

```diff
@@ -111,19 +111,20 @@ func TestFunctionApi(t *testing.T) {
 
 	m, err := g.Client().V1().Function().Create(testFunc)
 	panicIf(err)
-	defer func() {
-		err := g.Client().V1().Function().Delete(m)
-		panicIf(err)
-	}()
 
 	_, err = g.Client().V1().Function().Create(testFunc)
 	assertNameReuseFailure(err, "function")
 
 	testFunc.ObjectMeta.ResourceVersion = m.ResourceVersion
 	testFunc.Spec.Package.FunctionName = "yyy"
-	_, err = g.Client().V1().Function().Update(testFunc)
+	m, err = g.Client().V1().Function().Update(testFunc)
```

Comment:
Nit Can we please call it m3 or something unique instead of overriding m and then add `defer` for both of functions?